### ISSUE TITLE
How to specify per-layer learning rates in config file when TransferLearning?

### ISSUE THREAD

**Issue by W-Sword** [open-mmlab/mmdetection#3146]
After partially loading the pre-trained model, i need set different learning rates for different layers. The learning rate of the layer that can be loaded is set to 1e-6, and the learning rate of the layer that can not be loaded is set to 1e-4.

I guess i should modify the following code in config file,but i do not know the relational params name and format.

`optimizer = dict(type='SGD', lr=0.02, momentum=0.9, weight_decay=0.0001)`

Can somebody help me?Please give an example about it.

Thanks.

**Comment by ZwwWayne** [comment]
Hi @W-Sword ,
You need to re-implement an optimizer constructor inherit from the [default one](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/optimizer/default_constructor.py). With the constructor you can set learning rate of each layer in a fine-grained manner.

**Comment by hellock** [comment]
Specifying `cutom_keys` is feasible. See https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/optimizer/default_constructor.py#L18 for details.
